src/app/screens/Books/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
```

Comment:
index can be just a function instead of a class if you havent functions inside the "class"

Reply:
Done